### PR TITLE
[CI] hotfix broken tests on master

### PR DESCRIPTION
**Pull request by krivonogov**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -46,4 +46,3 @@ workflows:
     jobs:
       - "python-3.6"
       - "python-3.7"
-      - "upstream-dev"
```
Comment (krivonogov) [inline .circleci/config.yml:49]: I don't want to debug this problem for now: https://circleci.com/gh/krivonogov/fletcher/2?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: ci/circle_build_linux.sh [hunk 2]
```diff
@@ -22,7 +22,7 @@ conda config --add channels https://repo.continuum.io/pkgs/free
 conda config --add channels conda-forge
 
 conda create -y -q -n fletcher python=${PYTHON_VERSION} \
-    pandas pyarrow pytest pytest-cov pytest-flake8 \
+    pandas pyarrow=0.14 pytest pytest-cov pytest-flake8 \
```
Comment (krivonogov) [inline ci/circle_build_linux.sh:25]: there is a problem while using `pyarrow=0.15`: https://circleci.com/gh/krivonogov/fletcher/7?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

File: .circleci/config.yml (hunk 1 above)
Comment (artemru) [reply .circleci/config.yml:49]: https://issues.apache.org/jira/browse/ARROW-6844

File: ci/circle_build_linux.sh (hunk 2 above)
Comment (artemru) [reply ci/circle_build_linux.sh:25]: `0.14.1`?

Comment (krivonogov) [reply ci/circle_build_linux.sh:25]: conda should take last `0.14` available, so it will be `pyarrow-0.14.1` (I checked it in tests circleci output :) )

**Pull request merged**